lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -252,18 +252,16 @@ class AwsCompileStreamEvents {
               );
             }
 
+            streamResource.Properties.BisectBatchOnFunctionError = (event.stream.bisectBatchOnFunctionError === true);
```

Comment:
This will unconditionally set this property, and we don't want to do that (we have experience of side effects and sometimes breaking changes caused).

Let's just update the constraint in origin condition to `!= null`, and assign `event.stream.bisectBatchOnFunctionError` as it is

Reply:
@medikoo AWS default is false. Anyway i have updated this condition also.
Thanks @medikoo